### ISSUE TITLE
Add topic on DataMonitor

### ISSUE THREAD

**Issue by reinout538** [ubvu/open-handbook#250]
Elsevier will no longer support DataMonitor

* researchers will have to add datasets manually in Pure
* we will have to look into alternative options for automated import

**Comment by Elisa-on-GitHub** [comment]
@reinout538 I'm keeping this open for now, this seems like a real (i.e. not a test) issue, right?

**Comment by reinout538** [comment]
@Elisa-on-GitHub Yeah, it does look that way, but we need to get some more info / confirmation from Elsevier on this
